.travis.yml

```diff
@@ -94,6 +96,30 @@ matrix:
             - gcc-arm-linux-gnueabihf
             - libc6-dev-armhf-cross
 
+    - env: TARGET_X=arm-linux-androideabi CC_X=arm-linux-androideabi-gcc CXX_X=arm-linux-androideabi-g++ MODE_X=DEBUG KCOV=0
+      rust: stable
+      addons:
+        apt:
+          packages:
+            - expect
+            - openjdk-6-jre-headless
+
+    - env: TARGET_X=arm-linux-androideabi CC_X=arm-linux-androideabi-gcc CXX_X=arm-linux-androideabi-g++ MODE_X=RELWITHDEBINFO KCOV=0
+      rust: stable
+      addons:
+        apt:
+          packages:
+            - expect
+            - openjdk-6-jre-headless
+
+    - env: TARGET_X=arm-linux-androideabi CC_X=arm-linux-androideabi-gcc CXX_X=arm-linux-androideabi-g++ MODE_X=DEBUG KCOV=0
+      rust: nightly
```

Comment:
You are missing the `rust: nightly` + `MODE_X=RELWITHDEBINFO` option. Please add it.